src/Control/FunFlow/Exec/Redis.hs

```diff
@@ -16,78 +17,225 @@ import Control.Monad.State.Strict
 import Data.Monoid ((<>))
 import Control.Exception
 import Data.ByteString (ByteString)
+import qualified Data.ByteString as BS
+import qualified Data.ByteString.Char8 as BS8
 import qualified Database.Redis as R
+import Control.Monad.Except
+import GHC.Conc
+import GHC.Generics
+import Control.Concurrent.Async.Lifted
+import Control.Monad.Trans.Control
+import Control.Monad.Base
+import Lens.Micro.Platform
+import Control.FunFlow.Utils
```

Comment:
Likewise, style is to have imports alphabetically ordered. `stylish-haskell` will automatically order them for you.